.travis.yml

```diff
@@ -1,7 +1,22 @@
-sudo: false
+dist: trusty
+sudo: required
 language: node_js
 node_js:
 - '10'
+addons:
+  firefox: latest
+  apt:
+    sources:
+     - google-chrome
+    packages:
+     - google-chrome-stable fluxbox
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3
+  - fluxbox >/dev/null 2>&1 &
+before_install:
+  - stty cols 80
```

Comment:
curious what this is for

Reply:
I'm not sure. I think it has something to do with unix shell settings. it's from the testcafe documentations